youtube_dl/extractor/vlive.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import hmac
+from hashlib import sha1
+from base64 import b64encode
+from time import time
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    determine_ext
+)
+from ..compat import compat_urllib_parse
+
+
+class VLiveIE(InfoExtractor):
+    IE_NAME = 'vlive'
+    _VALID_URL = r'https?://(?:(www|m)\.)?vlive\.tv/video/(?P<id>[0-9]+)'
```

Comment:
They used to redirect http://www.vlive.tv/video/1326 to http://m.live.tv/video/1326 for all requests, but it seems that they only do the redirect for mobile devices now (you can try with your phone, or toggle device mode in Chrome's Developer Tools). Not sure if this change in behaviour is a temporary bug.

When you share a link from the V app to Twitter, the link generated is still http://vlive.tv/video/1326 which redirects to http://www.vlive.tv/video/1326.

I'm inclined to leave www in the url regex for this.